src/libmongoc/tests/unified/runner.c

```diff
@@ -1093,7 +1104,7 @@ test_check_event (test_t *test,
       bson_val_t *expected_val = bson_val_from_bson (expected_reply);
       bson_val_t *actual_val = bson_val_from_bson (actual->reply);
       if (!entity_map_match (
-             test->entity_map, expected_val, actual_val, true, error)) {
+             test->entity_map, expected_val, actual_val, false, error)) {
```

Comment:
This technically fixes [CDRIVER-4329](https://jira.mongodb.org/browse/CDRIVER-4329), which I folded into [CDRIVER-4374](https://jira.mongodb.org/browse/CDRIVER-4374).